new_tests/input/obsResidual.xml

```diff
@@ -0,0 +1,107 @@
+      <submitter>
+        <name>D. J. Tholen</name>
+        <institution>University of Hawaii</institution>
+      </submitter>
+      <observers>
+        <name>D. Hung</name>
+      </observers>
+      <measurers>
+        <name>D. J. Tholen</name>
+      </measurers>
+      <telescope>
+        <name>University of Hawaii 2.24-m</name>
+        <design>reflector</design>
+        <aperture>2.24</aperture>
+        <detector>STA1600 10k CCD</detector>
+        <fRatio>10.1</fRatio>
+        <filter>none</filter>
+        <arraySize>2112 x 2112 pixels</arraySize>
+        <pixelScale>0.4118</pixelScale>
+      </telescope>
+      <software>
+        <astrometry>Tholen custom ADES software, 2021 Apr 10 version</astrometry>
+        <fitOrder>1</fitOrder>
+      </software>
+      <fundingSource>NASA 80NSSC21K0807</fundingSource>
+      <comment>
+        <line>Second opposition recovery of 2022 KW.</line>
+        <line>Time correction was also applied to the field asteroid.</line>
+      </comment>
+    </obsContext>
+    <obsData>
+      <optical>
+        <provID>2022 KW</provID>
+        <mode>CCD</mode>
+        <stn>T12</stn>
+        <obsTime>2023-06-15T13:44:30.18Z</obsTime>
+        <ra>325.589699</ra>
+        <dec>+0.560433</dec>
+        <rmsRA>0.033</rmsRA>
+        <rmsDec>0.033</rmsDec>
+        <astCat>Gaia2</astCat>
+        <mag>22.458</mag>
+        <rmsMag>0.069</rmsMag>
+        <band>G</band>
+        <photCat>Gaia2</photCat>
+        <photAp>1.2</photAp>
+        <logSNR>1.20</logSNR>
+        <seeing>1.3</seeing>
+        <exp>420.0</exp>
+        <rmsFit>0.049</rmsFit>
+        <nStars>395</nStars>
+        <remarks>S: 0.0018 0.0031 O: 0.0328 0.0328</remarks>
+        <!-- Starting OpticalResidual sub-block -->
+                        <orbProd>Someone</orbProd>
+                        <orbID>1</orbID>
+        <!-- Starting OpticalRes sub-sub-block -->
+                                  <resRA>1.2</resRA>
+                                  <resDec>1.2</resDec>
+                                  <selAst>A</selAst>
+                                  <sigRA>1.2</sigRA>
+                                  <sigDec>1.2</sigDec>
+                                  <sigCorr>0.5</sigCorr><!-- optional -->
+                                  <sigTime>0.5</sigTime><!-- optional -->
+                                  <biasRA>1.2</biasRA><!-- optional -->
+                                  <biasDec>1.2</biasDec><!-- optional -->
+                                  <biasTime>1.2</biasTime><!-- optional -->
+        <!-- Starting OpticalResMag sub-sub-block -->
+                                  <photProd>Someone else</photProd><!-- optional -->
+                                  <resMag>1.2</resMag>
+                                  <selPhot>A</selPhot>
+                                  <sigMag>1.2</sigMag>
+                                  <biasMag>1.2</biasMag>
+                                  <photMod>abc123</photMod>
```

Comment:
Not a big problem, but the formatting looks weird to me. Is this how it's supposed to look like?